### PR TITLE
Pin lxml

### PR DESCRIPTION
**Pull request by ocefpaf**
This will fail on Windows because we do not have a conda package for `lxml 3.6.2` there yet.

See https://github.com/geopython/pycsw/issues/454

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,8 +36,10 @@ test:
 about:
     home: https://github.com/geopython/pycsw
     license: MIT
-    summary: OGC Catalog Service for the Web (CSW) implementation in python
+    summary: 'OGC Catalog Service for the Web (CSW) implementation in python.'
 
 extra:
     recipe-maintainers:
+        - kalxas
+        - tomkralidis
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @kalxas and @tomkralidis is that OK? Once merged you will get commit rights to this feedstock and will be able to merge PRs here.

For more details on what does being a conda-forge feedstock maintainer entail see https://github.com/conda-forge/staged-recipes#10-what-does-being-a-conda-forge-feedstock-maintainer-entail

**Comment by kalxas** [comment]
Thanks @ocefpaf

**Comment by ocefpaf** [comment]
We are at a loss on Windows! There is not even a `PyPI` wheel to re-package for `lxml 3.6.2`.

https://pypi.python.org/pypi/lxml

Hopefully we can do solve https://github.com/conda-forge/libxslt-feedstock/issues/1 and https://github.com/conda-forge/lxml-feedstock/issues/1 soon.

Merging this without Windows to fix it for `OS X` and `Linux` at least.

**Comment by kwilcox** [comment]
@ocefpaf this seems to have regressed?

```
$ conda install pycsw
Fetching package metadata .............
Solving package specifications: ..........

Package plan for installation in environment /home/kwilcox/miniconda3/envs/pycsw35:

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    geos-3.6.0                 |                0        19.9 MB  conda-forge
    shapely-1.5.16             |           py35_0         524 KB  defaults
    pyproj-1.9.5.1             |           py35_0         3.3 MB  defaults
    pycsw-2.0.2                |           py35_1         379 KB  conda-forge
    ------------------------------------------------------------
                                           Total:        24.1 MB

The following NEW packages will be INSTALLED:

    geolinks:        0.2.0-py35_0   conda-forge
    geos:            3.6.0-0        conda-forge
    icu:             56.1-4         conda-forge
    libiconv:        1.14-4         conda-forge
    libpq:           9.5.4-3        conda-forge
    libxml2:         2.9.3-9        conda-forge
    libxslt:         1.1.29-1       conda-forge
    lxml:            3.6.4-py35_1   conda-forge
    owslib:          0.13.0-py35_0  conda-forge
    psycopg2:        2.6.2-py35_1   conda-forge
    pycsw:           2.0.2-py35_1   conda-forge
    pyproj:          1.9.5.1-py35_0 defaults   
    python-dateutil: 2.6.0-py35_0   conda-forge
    pytz:            2016.7-py35_0  conda-forge
    requests:        2.12.1-py35_0  conda-forge
    shapely:         1.5.16-py35_0  defaults   
    six:             1.10.0-py35_1  conda-forge
    sqlalchemy:      1.1.4-py35_0   defaults   
    xmltodict:       0.10.2-py35_0  conda-forge
```

**Comment by kwilcox** [comment]
Ah. I see it was unpinned in favor of a pathed libxml2. Nothing to see here.

**Pull request merged**